v2/common/src/main/java/com/google/cloud/teleport/v2/cdc/mappers/MergeInfoMapper.java

```diff
@@ -68,8 +69,10 @@
 
   public MergeInfoMapper(
       DataStreamClient dataStreamClient,
+      String projectId,
```

Comment:
Could you please reformat the parameters so that only one appears per line? (I know it isn't your fault, but the inconsistency is becoming more apparent, and the one-per-line is better aligned with how we'll enforce formatting in the future.)